### ISSUE TITLE
Support RDM (E1.33)

### ISSUE THREAD

**Issue by k-yle** [k-yle/sACN#1]
Support _RDMNet_ using the new E1.33 protocol

See also: https://github.com/ETCLabs/RDMnet

**Comment by k-yle** [comment]
this is waiting on any major console vendor  to implement the new standard so that we can actually try it out.

**Comment by stephanschulz** [comment]
thanks for make this code public.
has RDM been implemented yet?

**Comment by k-yle** [comment]
Hi, it's been nearly 4 years and there's still very limited support for RDMNet across products and consoles, which makes this tricky to test... I've written a basic implementation of the LLRP & RPT protocols, but it's far from spec compliant and needs to be tested with some real equipment...

**Comment by stephanschulz** [comment]
Hi. Thanks for your reply.
I have an Enttec Storm8 artnet device https://www.enttec.com/product/uncategorized/storm-8-8-universe-ethernet-to-dmx-gateway/
and send DMX to these dimmers https://www.sunricher.com/din-rail-mounted-dmx512-decoder-sr-2103din.html#product_tabs_resources

Until now I always set their addresses by hand, but someone showed me how they were able to use a grandMA console https://www.malighting.com/grandma3/ to address all daisy chained dimmers in a sequential order. I assume that was done via RDM.

I am hoping to find a way to add this functionally of setting parameters like address, PWM frequency etc via RDM.

So, if you need someone to test code I am happy to give it a try :)

**Comment by k-yle** [comment]
As far as I'm aware, grandMA supports "RDM over Art-Net", but not RDMNet. I'll ask around if this is still the case

**Comment by stephanschulz** [comment]
I am still learning about RDM.
I did not realize RDMNet is different from regular RDM over artnet. 
In my case I am just interested in sending regular RDM messages to the dmx devices attached to the Entice storm8. It does not need to be RDMnet if that complicates things. Sorry for having maybe added to the wrong issue.

**Comment by k-yle** [comment]
yeah, RDMNet can be thought of as 'RDM over sACN' (this isn't quite true but it's a useful analogy)

I'm not aware of any javascript implementation of RDM over Art-Net, but the folks over at https://github.com/hobbyquaker/artnet/issues/25 might be able to point you in the right direction.

This goal of this paticular library is just to support sACN & RDMNet. I don't plan on adding Art-Net support unless both [hobbyquaker/artnet](https://github.com/hobbyquaker/artnet) and [margau/dmxnet](https://github.com/margau/dmxnet) become unmaintained, which would be a shame for the JS community
